src/wallet/wallet.cpp

```diff
@@ -542,6 +542,7 @@ void CWallet::SetMinVersion(enum WalletFeature nVersion, WalletBatch* batch_in)
     LOCK(cs_wallet);
     if (nWalletVersion >= nVersion)
         return;
+    this->WalletLogPrintf("Setting minversion to %d\n", nVersion);
```

Comment:
`this->` is unneeded.

Reply:
@achow101 I pushed a revision without it (I actually had the patch ready since this afternoon, but I forgot to push it until now).